GliaWidgets/Component/CallButtonBar/Button/CallButtonStyle.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 
 /// Style of a button shown in call view bottom button bar (i.e. "Chat", "Video", "Mute", "Speaker" and "Minimize").
 public struct CallButtonStyle {
-    /// Style of a call button ("Chat", "Video", "Mute", "Speaker" and "Minimize") in a specific state - active or inactive.
+    /// Style of a call button ("Chat", "Video", "Mute", "Speaker" and "Minimize") in a specific state - activated or not activated.
```

Comment:
There was a confusion when we first wrote the documentation. Inactive does not mean that the button is disabled. To indicate that, the app takes an inactive state and makes it more transparent.
Instead it means whether the button is toggled on or off. This only relates to the Mute button and the Speaker button - if you are muted, the "Mute" button is in "active" state, i.e. it is toggled on; same for the speaker.